README.md

```diff
@@ -42,6 +42,17 @@ Run tests
 Run tests with coverage:
 - `python -m pytest --cov=gdc_client --cov-branch --cov-report term tests/`
 
+## Install `pre-commit`
+
+This repository makes use of `pre-commit` for code formatting.
+In order to make use of it, run the following command:
+```
+pre-commit install
+```
```

Comment:
This README is laid out differently than gdcapi, which has a list of steps that fulfill pre-requisites for `pre-commit`.  For this repo (and possibly other repos), let's make this step more explicit:
```
pip install -r dev-requirements.txt
pre-commit install
```